core-web/libs/block-editor/src/lib/components/dot-block-editor/dot-block-editor.component.scss

```diff
@@ -5,6 +5,12 @@
 @import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;0,500;0,700;1,300;1,400&display=swap");
 @import url("https://fonts.googleapis.com/icon?family=Material+Icons");
 
+:host ::ng-deep {
+    @import "libs/dotcms-scss/angular/styles";
```

Comment:
This is problematic... because this will include all this code multiples times, I think we can do something let's check.

Reply:
We did research together, for now this is the best approach.